assets/js/modules/analytics/dashboard/dashboard-widget.js

```diff
@@ -174,11 +175,11 @@ class AnalyticsDashboardWidget extends Component {
 							{ ! receivingData && (
 								error ? getDataErrorComponent( _x( 'Analytics', 'Service name', 'google-site-kit' ), error, true, true, true ) : getNoDataComponent( _x( 'Analytics', 'Service name', 'google-site-kit' ), true, true, true )
 							) }
-							<div className={ `
+							<div className={ classnames( `
```

Comment:
Each of these strings should be there own string and line, not a multi-line template string.